service/service.go

```diff
@@ -158,6 +168,7 @@ func New(config Config) (*Service, error) {
 
 func (s *Service) Boot() {
 	s.bootOnce.Do(func() {
-		s.CustomObjectFramework.Boot()
+		go s.CRDFramework.Boot()
+		go s.CustomObjectFramework.Boot()
```

Comment:
Here we boot the different frameworks asynchronously so they do not block each other.